packages/keystone/src/fields/types/integer/tests/test-fixtures.ts

```diff
@@ -4,6 +4,7 @@ export const name = 'Integer';
 export const typeFunction = integer;
```

Comment:
Integer should need updating so that should happen + given the tests aren't failing because of it, I'm guessing something is wrong with the tests

Reply:
Looks like it's missing the `non-null` test fixtures 👍